modules/sticky-table-headers/main/index.coffee

```diff
@@ -63,7 +62,18 @@ getChildren = (parent, selector, single) ->
 createStickyHeaderNodes = (real, cloned) ->
   for i in [0...real.length]
     cloneEvents(real[i], cloned[i])
+    # The real table shouldn't show the sticky header nodes
     hx.select(real[i]).classed('hx-sticky-table-invisible', true)
+    hx.select(cloned[i]).classed('hx-sticky-table-invisible', false)
+
+cloneTableAndNodeEvents = (selection, realTable, tableClone, body, single) ->
+  innerTableClone = selection.append tableClone.clone(true)
+  # We make all the cells invisible by default and then only make the ones we care about visible in `createStickyHeaderNodes`
+  innerTableClone.selectAll('th, td').classed('hx-sticky-table-invisible', true)
```

Comment:
The key change is here, where we make _all_ the cells invisible in the 'cloned' table and then in `createStickyHeaderNodes` make only the ones we care about visible again.